ansible/Dockerfile

```diff
@@ -1,5 +1,23 @@
 FROM alpine:3.6
-RUN apk update
-RUN apk add git \
-    ansible
-RUN echo "'Done !"
\ No newline at end of file
+
+ARG CI_HELPER_VERSION
+
+WORKDIR /root
+RUN apk update &&\
+ apk add -q --no-cache 
+RUN apk add bash 
+RUN apk add ca-certificates 
+RUN apk add curl 
+RUN apk add git 
+RUN apk add openssh-client 
+RUN apk add python 
+RUN apk add py-pip 
+RUN apk add ansible
+RUN echo "Install CI Helper" && \
+    curl -sSL https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/sgerrand.rsa.pub -o /etc/apk/keys/sgerrand.rsa.pub && \
```

Comment:
There's [now](https://github.com/sgerrand/alpine-pkg-glibc/releases/) a `2.26-r0` release, it's probably better to get the latest.